weblate/machinery/forms.py

```diff
@@ -310,3 +310,13 @@ class OpenAIMachineryForm(KeyMachineryForm):
         ),
         required=False,
     )
+    base_url = forms.URLField(
+        label=pgettext_lazy(
+            "Automatic suggestion service configuration",
+            "OpenAI base URL",
+        ),
+        widget=forms.Textarea,
+        help_text=gettext_lazy("Base URL of the OpenAI API"),
+        required=False,
+        default=None,
```

Comment:
TypeError: Field.__init__() got an unexpected keyword argument 'default'

See https://github.com/WeblateOrg/weblate/actions/runs/8800026958/job/24150336022?pr=11467

Seriously, has this been tested to wokr?

Reply:
@nijel No, I just pushed it to see the pipeline! Not ready for review! I should have marked it as a draft!

Thanks for the fast review though.

Reply:
I recommend testing locally first against real custom API endpoint. The model selection will need to be changed for custom LLMs as well.